src/Popup.tsx

```diff
@@ -240,7 +250,10 @@ class Popup extends Component<PopupProps, PopupState> {
   }
 
   cancelFrameState = () => {
-    raf.cancel(this.nextFrameId);
+    if (this.nextFrameId) {
+      raf.cancel(this.nextFrameId);
+      this.nextFrameId = null;
+    }
```

Comment:
No need to add this check since cancel will not do anything if already finished or cancelled.

Reply:
OK, just undid that change